### PR TITLE
Restructured storage

### PR DESCRIPTION
**Pull request by jordan-powers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
 # Scouting Website
 https://momentum4999.com/scouting
 Contact a mentor for ftp access or cpanel access if you want to make changes.
+
+# Hidden sites
+Navigate to these sites by entering the url manually.
+- editschema.php : Edit the schema of data collected for each year
+- editusers.php : Manage users
```
Comment (terbos90803) [inline README.md:8]: Are these hidden pages protected in any way other than obscurity?

**Review by terbos90803 — COMMENTED** [review]
I didn't have time to go through the whole thing.  Let me know if there are any specific areas you'd like to talk through.

File: README.md (hunk 1 above)
Comment (jordan-powers) [reply README.md:8]: Yeah, you need to be logged in, and you need to have the 'admin' flag set on your account before you can access those pages. I just couldn't find a good place to link to them.

**Comment by jordan-powers** [comment]
The restructured storage seems to work, I'm going to merge it back to master

**Pull request merged**